tools/fpgabist/bist_common.py

```diff
@@ -93,6 +94,7 @@ class BistMode(object):
     name = ""
     executables = {}
     dir_path = ""
+    cwd = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I think you should rename `cwd`.  It sounds like *current working directory* but in fact it is the path to the script that you are executing, which might not be the actual working directory of the script.  Not sure of the best name, perhaps `opae_tools_path` (although that name is rather long).